cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/DatasetFrameworkTestUtil.java

```diff
@@ -47,10 +47,15 @@
 public final class DatasetFrameworkTestUtil extends ExternalResource {
   public static final Id.Namespace NAMESPACE_ID = Id.Namespace.from("myspace");
 
+  private Injector injector;
   private CConfiguration cConf;
   private TemporaryFolder tmpFolder;
   private DatasetFramework framework;
 
+  public Injector getInjector() {
```

Comment:
I don't think we need this change. Since we don't use this in any unit test, atleast I did not see any usage of `getInjector()` in this PR